tempesta_fw/http_match.c

```diff
@@ -64,6 +64,14 @@
 #include "http_match.h"
 #include "http.h"
 
+/* Matching every request against a long table of rules may produce a lot of
+ * debugging messages, so they are turned off by default. */
+#ifdef DEBUG_HTTP_MATCH
+#define MATCH_DBG(...) TFW_DBG(__VA_ARGS__)
+#else
+#define MATCH_DBG(...)
+#endif
+
```

Comment:
Oh, custom debug messages again, ever in main core module. This is messy to have own debug infrastructure for each C file.